sable_network/src/sync/replicated_log.rs

```diff
@@ -275,8 +275,21 @@ impl ReplicatedEventLog {
         &self,
         sender: UnboundedSender<Request>,
     ) -> JoinHandle<NetworkResult> {
+        let mut attempts = 0;
         while let Some(peer) = self.net.choose_any_peer() {
-            tracing::debug!("Requesting network state from {:?}", peer);
+            attempts += 1;
+            if attempts >= 3 {
+                tracing::info!(
+                    "Requesting network state from {:?} (attempt #{}).",
+                    peer,
+                    attempts
+                );
+                if attempts % 5 == 3 {
+                    tracing::warn!("Make sure at least one sable_ircd in your network is started and reachable. If this is your first sable_ircd process, you must provide the --bootstrap-network option.");
```

Comment:
```suggestion
                    tracing::warn!("Make sure at least one node in your network is started and reachable. If this is the first (or only) node, you must provide the --bootstrap-network option.");
```

The bootstrap node doesn't have to be an ircd; it could be services or any of the other specialised nodes I haven't written yet